src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -82,10 +83,8 @@ mutable struct AdditiveRungeKutta{
     rhs!::Any
     "rhs linear operator"
     rhs_implicit!::Any
-    "backward Euler solver"
-    besolver!::BE
-    "backward Euler solvers, pre-factorized, for MIS"
-    besolvers!::Any
+    "a dictionary of backward Euler solvers"
+    implicit_solvers::Any
```

Comment:
Do we really want an `Any` here? (also saying `Any` isn't needed since its implied)

Reply:
I think the formatter might apply `Any`, but I agree that more concrete is better here.